operator/gitops/argocd/pipeline-service/tekton-chains/chains-secrets-migration.yaml

```diff
@@ -1,192 +0,0 @@
----
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: tekton-chains-secrets-migrator
-  namespace: openshift-pipelines
-  annotations:
-    argocd.argoproj.io/sync-wave: "0"
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: tekton-chains-secret-migration
-  annotations:
-    argocd.argoproj.io/sync-wave: "0"
-rules:
-  - apiGroups:
-      - ""
-    resources:
-      - secrets
-    resourceNames:
-      - public-key
-      - signing-secrets
-    verbs:
-      - get
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: tekton-chains-secrets-migrator
-  namespace: tekton-chains
-  annotations:
-    argocd.argoproj.io/sync-wave: "0"
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: tekton-chains-secret-migration
-subjects:
-  - kind: ServiceAccount
-    name: tekton-chains-secrets-migrator
-    namespace: openshift-pipelines
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  name: openshift-pipelines-secret-migration
-  namespace: openshift-pipelines
-  annotations:
-    argocd.argoproj.io/sync-wave: "0"
-rules:
-  - apiGroups:
-      - ""
-    resources:
-      - secrets
-    verbs:
-      - list
-      - create
-      - get
-      - update
-      - patch
-      - delete
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
-metadata:
-  name: openshift-pipelines-secret-migration
-  namespace: openshift-pipelines
-  annotations:
-    argocd.argoproj.io/sync-wave: "0"
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: openshift-pipelines-secret-migration
-subjects:
-  - kind: ServiceAccount
-    name: tekton-chains-secrets-migrator
-    namespace: openshift-pipelines
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: openshift-gitops-jobs-admin
-  annotations:
-    argocd.argoproj.io/sync-wave: "0"
-rules:
-  - apiGroups:
-      - batch
-    resources:
-      - jobs
-    verbs:
-      - get
-      - list
-      - patch
-      - create
-      - delete
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: openshift-gitops-jobs-admin
-  annotations:
-    argocd.argoproj.io/sync-wave: "0"
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: openshift-gitops-jobs-admin
-subjects:
-  - kind: ServiceAccount
-    name: openshift-gitops-argocd-application-controller
-    namespace: openshift-gitops
----
-apiVersion: batch/v1
-kind: Job
```

Comment:
@Roming22 - I believe you verified the contents of the secrets this Job migrated when you first introduced the Job, but do you think we need to make sure the secret in openshift-pipelines has not become out of sync since you first checked it, or do you think such a check in unnecessary?

@ramessesii2 fyi

thanks

Reply:
Hi Gabe,

> do you think we need to make sure the secret in openshift-pipelines has not become out of sync

I might have missed what you meant exactly there but here's what I know - the signing-secrets are now directly generated in the openshift-pipelines ns by chains, and we're still [creating/updating the public-key with those secrets](https://github.com/openshift-pipelines/pipeline-service/pull/751/files/16774cd7c8cd4f26b011f72b817584fbe1a46249#diff-31dec81702caa0d2f573347854fd6d2a346d170d139e77d02a19ef6b6d858ac7R78).

I'll wait for anything @Roming22 has to say about it.

Reply:
The secret was copied at migration. Since then only the secret in the `openshift-pipelines` has been used, and we have not rotated any secrets since.

Therefore I don't think a check is necessary.